packages/phone-number-privacy/combiner/test/signing/bls-signature.test.ts

```diff
@@ -189,11 +189,11 @@ describe(`BLS service computes signature`, () => {
 
     const blsCryptoClient = new BLSCryptographyClient()
     // Add sigs one-by-one and verify intermediary states
-    await blsCryptoClient.addSignature(signatures[0])
+    blsCryptoClient.addSignature(signatures[0])
```

Comment:
not awaiting changes the test here. since the expect will run before addSignature actually completes

Reply:
But an await here doesn't do anything from what I can tell. The `addSignature` method isn't async it's just pushing to a list.

Reply:
```
  public addSignature(serviceResponse: ServicePartialSignature) {
    this.unverifiedSignatures.push(serviceResponse)
  }
```